photo-helper/src/main/java/io/intrepid/photohelper/PhotoHelper.java

```diff
@@ -46,8 +48,51 @@
      * @param view               - The {@link io.intrepid.photohelper.PhotoContract.View View} interface (may very likely be the {@link Fragment})
      * @param removeUnusedPhotos - If true, delete unused new photos that the user takes with the camera
      */
-    public PhotoHelper(@NonNull Fragment fragment, @NonNull PhotoContract.View view, boolean removeUnusedPhotos) {
-        this.fragment = fragment;
+    public PhotoHelper(@NonNull final Fragment fragment, @NonNull PhotoContract.View view, boolean removeUnusedPhotos) {
+        this.contextDelegate = new ContextDelegate() {
+            @Override
+            public void requestPermissions(@NonNull String[] permissions, int requestCode) {
+                fragment.requestPermissions(permissions, requestCode);
+            }
+
+            @Override
+            public ContextWrapper getContext() {
+                return fragment.getActivity();
+            }
+
+            @Override
+            public void startActivityForResult(Intent intent, int requestCode) {
+                fragment.startActivityForResult(intent, requestCode);
+            }
+        };
+        this.view = view;
```

Comment:
Can you create a private PhotoHelper constructor that both public constructors call which takes the contextDelegate, view, and removeUnusedPhotos boolean, and does the common initialization there?